docs/source/user-guide-gc/intro.rst

```diff
@@ -1,20 +0,0 @@
-#################
-Configuring HEMCO
-#################
-
-Usage of HEMCO within GEOS-Chem is documented at the GEOS-Chem Wiki,
```

Comment:
I will pull some of this info over to the geos-chem.readthedocs.io page (which has to be updated to link to the new info on hemco.readthedocs.io).